Resources/Prototypes/Procedural/salvage_mods.yml

```diff
@@ -65,195 +51,184 @@
-  id: LavaBrig
-  proto: LavaBrig
-  biomes:
-    - Lava
-
-# Air mixtures
-- type: salvageAirMod
-  id: Space
-  desc: No atmosphere
-  space: true
   cost: 1
-  biomes:
-  - Caves
-  - Lava
-
-- type: salvageAirMod
-  id: Breathable
-  gases:
-  - 21.824779 # oxygen
-  - 82.10312 # nitrogen
-  biomes:
-  - Caves
-  #- LowDesert
-  - Snow
-  - Grasslands
-
-- type: salvageAirMod
-  id: Sleepy
-  cost: 1
-  desc: Dangerous atmosphere
-  gases:
-  - 21.824779 # oxygen
-  - 72.10312 # nitrogen
-  - 0
-  - 0
-  - 0
-  - 0
-  - 0
-  - 10 # nitrous oxide
-  biomes:
-  - Caves
-  #- LowDesert
-  - Snow
-  - Grasslands
-  - Lava
-
-- type: salvageAirMod
-  id: Poisoned
-  cost: 2
-  desc: Dangerous atmosphere
-  gases:
-  - 21.824779 # oxygen
-  - 77.10312 # nitrogen
-  - 10 # carbon dioxide
-  biomes:
-  - Caves
-  #- LowDesert
-  - Snow
-  - Grasslands
-  - Lava
-
-- type: salvageAirMod
-  id: Poison
-  cost: 3
-  desc: Toxic atmosphere
-  gases:
-  - 21.824779 # oxygen
-  - 0
-  - 82.10312 # carbon dioxide
-  biomes:
-  - Caves
-  - Snow
-  - Lava
-
-- type: salvageAirMod
-  id: Plasma
-  cost: 4
-  desc: Toxic atmosphere
-  gases:
-  - 0
-  - 0
-  - 0
-  - 103.927899 # plasma
-  biomes:
-  - Caves
-  - Lava
-
-- type: salvageAirMod
-  id: Burnable
-  cost: 5
-  desc: Volatile atmosphere
-  gases:
-  - 21.824779 # oxygen
-  - 0
-  - 0
-  - 82.10312 # plasma
-  biomes:
-  - Caves
-  - Lava
+  color: null
 
 # Temperatures
-
 - type: salvageTemperatureMod
   id: RoomTemp
-  biomes:
-  - Caves
-  #- LowDesert
-  - Grasslands
+  cost: 0
 
 - type: salvageTemperatureMod
   id: Hot
+  cost: 1
   temperature: 323.15 # 50C
   biomes:
-  - Caves
-  #- LowDesert
-  - Grasslands
-  - Lava
+    - Caves
+    #- LowDesert
+    - Grasslands
+    - Lava
 
 - type: salvageTemperatureMod
   id: Burning
   desc: High temperature
-  cost: 1
+  cost: 2
   temperature: 423.15 # 200C
   biomes:
-  - Caves
-  #- LowDesert
-  - Lava
+    - Caves
+    #- LowDesert
+    - Lava
 
 - type: salvageTemperatureMod
   id: Melting
   desc: Extreme heat
   cost: 4
   temperature: 1273.15 # 1000C hot hot hot
   biomes:
-  - Lava
+    - Lava
 
 - type: salvageTemperatureMod
   id: Cold
+  cost: 1
   temperature: 275.15 # 2C
   biomes:
-  - Caves
-  #- LowDesert
-  - Grasslands
-  - Snow
+    - Caves
+    #- LowDesert
+    - Grasslands
+    - Snow
 
 - type: salvageTemperatureMod
   id: Tundra
   desc: Low temperature
   cost: 2
   temperature: 263.15 # -40C
   biomes:
-  - Caves
-  - Snow
+    - Caves
+    - Snow
 
 - type: salvageTemperatureMod
   id: Frozen
   desc: Extreme cold
-  cost: 3
+  cost: 4
   temperature: 123.15 # -150C
   biomes:
-  - Snow
+    - Snow
+
+# Air mixtures
+- type: salvageAirMod
+  id: Space
+  desc: No atmosphere
+  space: true
+  cost: 2
+  biomes:
+    - Caves
+    - Lava
+
+- type: salvageAirMod
+  id: Breathable
+  cost: 0
+  gases:
+    - 21.824779 # oxygen
+    - 82.10312 # nitrogen
+
+# TODO: These are good but mobs don't entirely handle it yet.
```

Comment:
only atmos i see mobs having issues with is high temperatures, they dont breathe plasma or nitrous oxide so they dont get affected

Reply:
sure.........